### DISCUSSION TITLE
Show links when deleting a page

### DISCUSSION THREAD

**Discussion by Keanegdom** [GitbookIO/community#634] [0. General bugs and questions] [answered]
**I think GitBook used to show a list of pages that contain a link to the page you are trying to delete.** 
It was great for making sure you don't break any references when deleting pages.
Now this feature seems to be gone. Is it a bug that's going to be resolved soon?

**Comment by valentin0h** [comment]
Hey @Keanegdom, 

We've temporarily disabled this feature in change requests for organizations not in the Pro or Enterprise plan. This is while we upgrade to a better solution for detecting links to pages and files.

The ability to detect links in the main content of a space (i.e not a change request) is still available to all plans. Let me know if that is not the case for you.

**Reply by Keanegdom** [reply]
Hi @valentin0h, thanks for your prompt reply!

Maybe it's due to my inexperience, but I don't think I follow. This is how I used to track references to a page:
1. In the edit mode, I pressed the "Delete" button next to a page.
2. I could see a list of all the pages containing links to the current page on the confirmation pop-up.

Recently, this option stopped working for me, as the list of pages is no longer in the pop-up.
Are you saying that there is another way of tracking references between pages? I would appreciate some guidance if, indeed, there is a way of doing it on the Personal or Plus plans.

**Reply by valentin0h** [reply, marked as answer]
Yes exactly, the _edit_ mode is a change request in GitBook. 

We have temporarily removed support for detecting references/links between pages in change requests for organizations in the Personal/Plus plans. We're looking into adding this back soon.
